File: app/src/main/java/com/evaluation/fragment/BaseFragment.java
package com.evaluation.fragment;

import android.app.Fragment;

/**
 * @author Vladyslav Havrylenko
 * @since 09.03.2020
 */
public class BaseFragment extends Fragment {
    public final String TAG = this.getClass().getCanonicalName();

    public boolean onBackPressed() {
        return true;
    }
}

File: app/src/main/java/com/evaluation/fragment/BaseFragment.kt
package com.evaluation.fragment

import androidx.fragment.app.Fragment

/**
 * @author Vladyslav Havrylenko
 * @since 09.03.2020
 */
open class BaseFragment : Fragment() {

    open val TAG = this.javaClass.canonicalName

    open fun onBackPressed(): Boolean = true
}